frontend/src/theme/useTheme.ts

```diff
@@ -4,10 +4,9 @@ import { atom, useAtomValue } from "jotai";
 import { userConfigAtom } from "@/core/config/config";
 import { isIslands } from "@/core/islands/utils";
 
-export type Theme = "light" | "dark" | "system";
-export type ResolvedTheme = "light" | "dark";
+export type Theme = "light" | "dark";
 
-export const THEMES: Theme[] = ["light", "dark", "system"];
+export const THEMES: Theme[] = ["light", "dark"];
```

Comment:
we want to keep system.

it will be tough for the backend to know, so lets just assume `system` gets resolved to `light` for now _just on the backend_. I have an idea for how the backend can get this information.